doc/holochain_101/src/nodejs_calling_zome_functions.md

```diff
@@ -0,0 +1,52 @@
+# Calling Zome Functions
+
+### `conductor.call(instanceId, zomeName, functionName, callParams)` => `object`
+
+A `Conductor` instance allows making calls to the custom functions defined the Zomes of DNA instances that it is running.
+This is necessary in order to be able to test them. It calls synchronously and returns the result that the Zome function provides. An error could also be thrown, or returned.
+
+Note that Holochain has to serialize the actual arguments for the
+function call into JSON strings, which the Conductor instance will handle for you automatically. It also parses the result from a JSON string into an object. If for some reason you don't want it to do that, do `conductor._callRaw` instead, and pass a string as the fourth argument instead of an object.
```

Comment:
```suggestion
function call into JSON strings, which the Conductor instance will handle for you automatically. It also parses the result from a JSON string into an object. If for some reason you don't want it to do that, do `conductor._callRaw` instead, and pass a string as the last argument instead of an object.
```

Only suggesting because it may be more robust to future changes